db/migrate/20230815111348_create_post.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :title
+      t.integer :commentsCounter
+      t.integer :likesCounter
+
+      t.timestamps
```

Comment:
- [ ] KIndly make sure to rename the column `author` instead of `user` in order to match the provided [ERD DIAGRAM](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) and project requirement.

```suggestion
    create_table :posts do |t|
      t.references :author, foreign_key: { to_table: 'users' }
      t.string :title
      t.integer :commentsCounter
      t.integer :likesCounter

      t.timestamps
```